lib/dispatcher.js

```diff
@@ -151,7 +151,7 @@ Dispatcher.prototype.dispatch = function(params, dispatchOptions) {
         me.authenticator.authenticateRequest(params);
       }
       params.headers = _.extend(
-          {}, me.defaultHeaders, dispatchOptions.headers || {});
+          {}, params.headers, me.defaultHeaders, dispatchOptions.headers || {});
```

Comment:
Then we set the default headers, overwriting anything so far - so someone can overwrite the Authentication: header if they want by passing it via dispatchOptions...

Reply:
That's true, but rare, and they should know what they're doing if that's what they want.